src/state/mod.rs

```diff
@@ -127,18 +159,22 @@ impl State {
     }
 
     pub fn input(&mut self, event: &WindowEvent) -> bool {
+        self.camera_controller.process_events(event);
```

Comment:
this method also returns boolean, why don't we take it into account when returning the boolean here? 🤔

Reply:
this will be revised in the future. The function is added from the tutorial.